tutorials/external-metastore-for-dataproc/index.md

```diff
@@ -0,0 +1,276 @@
+        LOCATION 'gs://${PROJECT}-warehouse/datasets/transactions';"
+
+### Create Worker clusters
+
+In this section, you create two Dataproc clusters in different regions preferrably in the same multi-region covered by the warehouse bucket.
+
+In Cloud Shell, run the following command to create first worker cluster in the same region:
+
+    gcloud dataproc clusters create hive-worker1 \
+    --image-version 2.0-debian10 \
+    --region ${REGION} \
+    --zone ${ZONE} \
+    --properties=^#^hive:hive.metastore.warehouse.dir=gs://${PROJECT}-warehouse/datasets \
+    --properties=^#^hive:hive.metastore.uris=thrift://hive-metastore1-m:9083,thrift://hive-metastore2-m:9083
+
+Note: The `hive.metastore.uris` property points the metastore service to the external Hive metastore service clusters. It is possible to indicate multiple, comma-separated metastore instances.
+
+## Running Hive Query on workers
+
+You can use different tools inside Dataproc to run Hive queries like Hive jobs API, [Beeline](https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-Beeline%E2%80%93CommandLineShell) CLI client and [SparkSQL](https://spark.apache.org/sql/). In this section, you learn how to perform queries using the Hive jobs API:
+
+    gcloud dataproc jobs submit hive \
+    --cluster hive-worker1 \
+    --region ${REGION} \
+    --execute "SELECT * FROM transactions LIMIT 10;"
+
+The output includes the following:
+
+    +-----------------+--------------------+------------------+
+    | submissiondate  | transactionamount  | transactiontype  |
+    +-----------------+--------------------+------------------+
+    | 2017-12-03      | 1167.39            | debit            |
+    | 2017-09-23      | 2567.87            | debit            |
+    | 2017-12-22      | 1074.73            | credit           |
+    | 2018-01-21      | 5718.58            | debit            |
+    | 2017-10-21      | 333.26             | debit            |
+    | 2017-09-12      | 2439.62            | debit            |
+    | 2017-08-06      | 5885.08            | debit            |
+    | 2017-12-05      | 7353.92            | authorization    |
+    | 2017-09-12      | 4710.29            | authorization    |
+    | 2018-01-05      | 9115.27            | debit            |
+    +-----------------+--------------------+------------------+
+
+## Creating another Dataproc cluster
+
+In this section, you create another Dataproc cluster to verify that the Hive data and Hive metastore can be shared across multiple clusters.
+
+1.  Create the second worker cluster in a different region:
+
+        gcloud dataproc clusters create hive-worker2 \
+        --image-version 2.0-debian10 \
+        --region ${REGION2} \
+        --zone ${ZONE2} \
+        --properties=^#^hive:hive.metastore.warehouse.dir=gs://${PROJECT}-warehouse/datasets \
+        --properties=^#^hive:hive.metastore.uris=thrift://hive-metastore1-m:9083,thrift://hive-metastore2-m:9083
```

Comment:
Fully qualified domain name (FQDN) is required per https://cloud.google.com/compute/docs/internal-dns 

`--properties=^#^hive:hive.metastore.uris=thrift://hive-metastore1-m.us-central1-a.c.dss-hive-meta-4.internal:9083,thrift://hive-metastore2-m.us-central1-a.c.dss-hive-meta-4.internal:9083`